tasks/fetch-cert.yml

```
@@ -13,6 +13,7 @@
 - set_fact:
     certificate: '{{ response.json.data.data.certificate }}'
     private_key: '{{ response.json.data.data.private_key }}'
+    expired: '{{ (response.json.data.data.expiration_date | default(1745829694) | int) < ( ansible_date_time.epoch |int)}}'
```

Why are you hardcoding `1745829694` as default? Is it because not all certs have an expiry date set?

Shouldn't we just add missing values? This should fail if it's missing right?

---

The default value is added for the case a certificate has been created and store before this version, it won't have the `expired` field. to avoid having a failure in date calculation i added a default value to make sure it's expired.